qml/main.qml

```diff
@@ -271,31 +271,31 @@ ApplicationWindow {
         id: shortcutCopy
         property string description: "Copy account"
         sequence: StandardKey.Copy
-        enabled: !!currentCredentialCard
+        enabled: isCurrentObjectName("authenticatorView") && !!currentCredentialCard
         onActivated: app.currentCredentialCard.calculateCard(true)
     }
 
     Shortcut {
         id: shortcutDelete
         property string description: "Delete account"
-        sequence: StandardKey.Delete
-        enabled: !!currentCredentialCard
+        sequence: Qt.platform.os === "osx" ? "Backspace" : "Delete"
```

Comment:
Why?

Reply:
StandardKey.Delete doesn't work on macOS for some reason.